File: ranlian-app-master/app/src/main/java/com/czsy/bean/YunShuDanBean.java
package com.czsy.bean;

import java.util.List;

public class YunShuDanBean implements IBaseBean {


    /**
     * {"id":"2","changZhanId":"3","changZhan":null,"gongYingZhanId":"4",
     * "gongYingZhan":null,"gangPingCount":null,"gangPingIdList":null,
     * "yunShuYuanId":"12","yunShuYuanName":"运输大队长","chePai":"",
     * "chuangJianRiQi":"2019-09-22 22:47:49","type":1,"typeName":"供应站-》厂站"}
     */
    public long id;
    public long  changZhanId;
    public String changZhan;
    public long gongYingZhanId;
    public Integer gangPingCount;
    public String gongYingZhan;
    //public List<Long> gangPingIdList;
    public long yunShuYuanId;
    public String yunShuYuanName;
    public String chePai; // 车牌
    public String chuangJianRiQi;
    public int type;
    public String typeName;
    public String statusName;


}

File: ranlian-app-master/app/src/main/java/com/czsy/bean/gongyingzhanzz/YunShuDanBean.kt
package com.czsy.bean.gongyingzhanzz

open class YunShuDanBean<T>:ArrayList<ItemBean>()

data class ItemBean(
    val changZhan: String,
    val changZhanId: String,
    val chePai: String,
    val chuangJianRiQi: String,
    val gangPingCount: Int,
    val gangPingIdList: Any,
    val gangPingTongJi: GangPingTongJi,
    val gongYingZhan: String,
    val gongYingZhanId: String,
    val id: String,
    val shenPiRen: String,
    val statistics: Statistics,
    val status: Int,
    val statusName: String,
    val type: Int,
    val typeName: String,
    val yuQi: Double,
    val yunShuYuanId: String,
    val yunShuYuanName: String
)

data class GangPingTongJi(
    val gangPing10KGCount: Int,
    val gangPing15KGCount: Int,
    val gangPing50KGCount: Int,
    val gangPing50KGIICount: Int,
    val gangPing50KGYeCount: Int,
    val gangPing5KGCount: Int,
    val shiShouYaJin: Int,
    val sum: Int,
    val xiaoShouE: Int
)

data class Statistics(
    val anJian: AnJian,
    val baoFei: BaoFei,
    val kongPing: KongPing,
    val zhongPing: ZhongPing
)

data class AnJian(
    val gangPing10KGCount: Int,
    val gangPing15KGCount: Int,
    val gangPing50KGCount: Int,
    val gangPing50KGIICount: Int,
    val gangPing50KGYeCount: Int,
    val gangPing5KGCount: Int,
    val shiShouYaJin: Int,
    val sum: Int,
    val xiaoShouE: Int
)

data class BaoFei(
    val gangPing10KGCount: Int,
    val gangPing15KGCount: Int,
    val gangPing50KGCount: Int,
    val gangPing50KGIICount: Int,
    val gangPing50KGYeCount: Int,
    val gangPing5KGCount: Int,
    val shiShouYaJin: Int,
    val sum: Int,
    val xiaoShouE: Int
)
